packages/blitz/src/renderer/render.rs

```diff
@@ -1091,4 +1093,61 @@ impl ElementCx<'_> {
     ) {
         unimplemented!()
     }
+
+    fn draw_input(&self, scene: &mut Scene) {
+        if self.element.local_name() == "input"
+            && matches!(self.element.attr(local_name!("type")), Some("checkbox"))
+        {
+            let checked: bool = self
+                .element
+                .attr(local_name!("checked"))
+                .and_then(|c| c.parse().ok())
+                .unwrap_or_default();
```

Comment:
`checked` is a boolean attribute, so this should just check for the presence of the `checked` attribute in the array. If it is present then `checked` is `true` else `false` (even the empty string or "false" counts as `true`).

See: https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML